### PR TITLE
feat: Add possibility to define feature_store.yaml path with env variable

### PR DESCRIPTION
**Pull request by arcabucero**
Signed-off-by: arcabucero <arcabucero@gmail.com>

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

File: sdk/python/feast/cli.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def cli(
     ctx.obj["FS_YAML_FILE"] = (
         Path(feature_store_yaml).absolute()
         if feature_store_yaml
-        else ctx.obj["CHDIR"] / "feature_store.yaml"
+        else utils.get_default_yaml_file_path(ctx.obj["CHDIR"])
```
Comment (felixwang9817) [inline sdk/python/feast/cli.py:102]: nice! I think you're missing a few spots in the codebase where you could also introduce this change

if you search for '"feature_store.yaml"' you should fine all of them, e.g. there's one in `infra/aws.py`

Comment (felixwang9817) [reply sdk/python/feast/cli.py:102]: also do you mind adding a test? I would take a look at `test_cli.py`, specifically you should be able to modify `test_3rd_party_registry_store_with_fs_yaml_override` to test your change pretty easily!

**Review by felixwang9817 — COMMENTED** [review]
thanks for the contribution @arcabucero, looks solid overall!

File: sdk/python/feast/cli.py (hunk 1 above)
Comment (arcabucero) [reply sdk/python/feast/cli.py:102]: @felixwang9817, thank you for your suggestions,
I've also introduced changes in aws.py and lambda_engine.py and added one test into test.cli.py

**Review by felixwang9817 — APPROVED** [review]
/lgtm

**Pull request merged**